--- dia-0.97.2/app/app_procs.c.0002~	2011-12-19 09:27:57.808631627 +0100
+++ dia-0.97.2/app/app_procs.c	2011-12-19 09:35:33.361501430 +0100
@@ -50,7 +50,7 @@
 #include <libxml/parser.h>
 #include <libxml/xmlerror.h>
 
-#include <glib/gstdio.h>
+#include <glib.h>
 
 #include "intl.h"
 #include "app_procs.h"
--- dia-0.97.2/app/dia-win-remote.c.0002~	2010-08-03 17:35:35.000000000 +0200
+++ dia-0.97.2/app/dia-win-remote.c	2011-12-19 09:35:33.341501436 +0100
@@ -35,7 +35,7 @@
 #include <shellapi.h>
 #include <Shlwapi.h>
 #include <glib.h>
-#include <glib/gprintf.h>
+#include <glib.h>
 
 /**
  * PROTOTYPES:
--- dia-0.97.2/app/filedlg.c.0002~	2009-11-07 18:13:53.000000000 +0100
+++ dia-0.97.2/app/filedlg.c	2011-12-19 09:35:33.317501440 +0100
@@ -28,7 +28,7 @@
 #include <unistd.h>
 #endif
 #include <stdio.h>
-#include <glib/gstdio.h>
+#include <glib.h>
 
 #undef GTK_DISABLE_DEPRECATED /* gtk_file_chooser_dialog_new_with_backend */
 #include <gtk/gtk.h>
--- dia-0.97.2/app/load_save.c.0002~	2011-09-25 13:55:11.000000000 +0200
+++ dia-0.97.2/app/load_save.c	2011-12-19 09:35:33.335501436 +0100
@@ -30,7 +30,7 @@
 #include <string.h>
 
 #include <glib.h>
-#include <glib/gstdio.h> /* g_access() and friends */
+#include <glib.h> /* g_access() and friends */
 #include <errno.h>
 
 #ifndef W_OK
--- dia-0.97.2/app/sheets_dialog_callbacks.c.0002~	2009-11-07 18:13:53.000000000 +0100
+++ dia-0.97.2/app/sheets_dialog_callbacks.c	2011-12-19 09:35:33.391501424 +0100
@@ -44,7 +44,7 @@
 #endif
 #endif
 
-#include <glib/gstdio.h>
+#include <glib.h>
 #include <gmodule.h>
 
 #undef GTK_DISABLE_DEPRECATED /* GtkOptionMenu */
--- dia-0.97.2/lib/debug.c.0002~	2009-11-07 15:28:34.000000000 +0100
+++ dia-0.97.2/lib/debug.c	2011-12-19 09:35:33.415501418 +0100
@@ -21,7 +21,7 @@
 #include <config.h>
 
 #include <stdio.h>
-#include <glib/gprintf.h>
+#include <glib.h>
 #include <stdarg.h>
 
 #include "debug.h"
--- dia-0.97.2/lib/dia_dirs.c.0002~	2011-12-19 09:27:57.821631622 +0100
+++ dia-0.97.2/lib/dia_dirs.c	2011-12-19 09:35:33.396501422 +0100
@@ -30,7 +30,7 @@
 #include <sys/stat.h>
 #include <sys/types.h>
 #endif
-#include <glib/gstdio.h>
+#include <glib.h>
 
 /** Get the name of a subdirectory of our data directory.
  *  This function does not create the subdirectory, just make the correct name.
--- dia-0.97.2/lib/dia_xml.c.0002~	2009-11-07 15:28:34.000000000 +0100
+++ dia-0.97.2/lib/dia_xml.c	2011-12-19 09:35:33.414501418 +0100
@@ -25,7 +25,7 @@
 #include <fcntl.h>
 
 #include <glib.h>
-#include <glib/gstdio.h>
+#include <glib.h>
 
 #include <libxml/parser.h>
 #include <libxml/parserInternals.h>
--- dia-0.97.2/objects/custom/shape_typeinfo.c.0002~	2009-11-07 15:28:34.000000000 +0100
+++ dia-0.97.2/objects/custom/shape_typeinfo.c	2011-12-19 09:35:33.279501450 +0100
@@ -27,8 +27,8 @@
 #include "custom_util.h"
 #include <string.h>
 #include <stdarg.h>
-#include <glib/gstrfuncs.h>
-#include <glib/gstdio.h>
+#include <glib.h>
+#include <glib.h>
 #include <libxml/parser.h>
 
 /*
--- dia-0.97.2/objects/SISSI/sissi.c.0002~	2009-11-07 15:28:34.000000000 +0100
+++ dia-0.97.2/objects/SISSI/sissi.c	2011-12-19 09:35:33.305501444 +0100
@@ -42,7 +42,7 @@
 #include "dia_xml_libxml.h"
 
 #include <string.h>
-#include <glib/gprintf.h>
+#include <glib.h>
 
 #define DEFAULT_WIDTH  1.0
 #define DEFAULT_HEIGHT 1.0
--- dia-0.97.2/objects/standard/image.c.0002~	2009-11-07 18:13:53.000000000 +0100
+++ dia-0.97.2/objects/standard/image.c	2011-12-19 09:35:33.290501447 +0100
@@ -25,7 +25,7 @@
 #ifdef HAVE_UNIST_H
 #include <unistd.h>
 #endif
-#include <glib/gstdio.h>
+#include <glib.h>
 
 #include "intl.h"
 #include "message.h"
--- dia-0.97.2/plug-ins/cairo/diacairo.c.0002~	2009-11-07 18:13:53.000000000 +0100
+++ dia-0.97.2/plug-ins/cairo/diacairo.c	2011-12-19 09:35:33.127501485 +0100
@@ -28,7 +28,7 @@
 #include <errno.h>
 #define G_LOG_DOMAIN "DiaCairo"
 #include <glib.h>
-#include <glib/gstdio.h>
+#include <glib.h>
 
 /*
  * To me the following looks rather suspicious. Why do we need to compile
--- dia-0.97.2/plug-ins/cairo/diacairo-renderer.c.0002~	2011-01-07 12:54:21.000000000 +0100
+++ dia-0.97.2/plug-ins/cairo/diacairo-renderer.c	2011-12-19 09:35:33.119501487 +0100
@@ -28,7 +28,7 @@
 #include <errno.h>
 #define G_LOG_DOMAIN "DiaCairo"
 #include <glib.h>
-#include <glib/gstdio.h>
+#include <glib.h>
 
 #ifdef HAVE_PANGOCAIRO_H
 #include <pango/pangocairo.h>
--- dia-0.97.2/plug-ins/cgm/cgm.c.0002~	2009-12-27 17:22:38.000000000 +0100
+++ dia-0.97.2/plug-ins/cgm/cgm.c	2011-12-19 09:35:33.077501497 +0100
@@ -31,7 +31,7 @@
 #include <glib.h>
 #include <errno.h>
 
-#include <glib/gstdio.h>
+#include <glib.h>
 
 #include "intl.h"
 #include "message.h"
--- dia-0.97.2/plug-ins/drs/dia-render-script.c.0002~	2009-11-07 18:13:53.000000000 +0100
+++ dia-0.97.2/plug-ins/drs/dia-render-script.c	2011-12-19 09:35:32.949501527 +0100
@@ -54,7 +54,7 @@
 
 #define G_LOG_DOMAIN "DiaRenderScript"
 #include <glib.h>
-#include <glib/gstdio.h>
+#include <glib.h>
 
 #include "intl.h"
 #include "filter.h"
--- dia-0.97.2/plug-ins/dxf/dxf-export.c.0002~	2009-11-07 15:28:34.000000000 +0100
+++ dia-0.97.2/plug-ins/dxf/dxf-export.c	2011-12-19 09:35:32.974501521 +0100
@@ -29,7 +29,7 @@
 #include <math.h>
 #include <glib.h>
 #include <errno.h>
-#include <glib/gstdio.h>
+#include <glib.h>
 
 #include "autocad_pal.h"
 
--- dia-0.97.2/plug-ins/dxf/dxf-import.c.0002~	2009-11-07 15:28:34.000000000 +0100
+++ dia-0.97.2/plug-ins/dxf/dxf-import.c	2011-12-19 09:35:32.996501516 +0100
@@ -30,7 +30,7 @@
 #include <string.h>
 #include <math.h>
 #include <glib.h>
-#include <glib/gstdio.h>
+#include <glib.h>
 
 #include "intl.h"
 #include "message.h"
--- dia-0.97.2/plug-ins/hpgl/hpgl.c.0002~	2009-11-07 15:28:34.000000000 +0100
+++ dia-0.97.2/plug-ins/hpgl/hpgl.c	2011-12-19 09:35:33.048501503 +0100
@@ -37,7 +37,7 @@
 #include <errno.h>
 
 #include <glib.h>
-#include <glib/gstdio.h>
+#include <glib.h>
 
 #include "intl.h"
 #include "message.h"
--- dia-0.97.2/plug-ins/libart/export_png.c.0002~	2011-07-03 12:56:08.000000000 +0200
+++ dia-0.97.2/plug-ins/libart/export_png.c	2011-12-19 09:35:33.036501507 +0100
@@ -29,7 +29,7 @@
 #include <errno.h>
 #include <stdlib.h>
 
-#include <glib/gstdio.h>
+#include <glib.h>
 #include <gtk/gtk.h>
 
 #include "intl.h"
--- dia-0.97.2/plug-ins/metapost/render_metapost.c.0002~	2009-11-07 15:28:34.000000000 +0100
+++ dia-0.97.2/plug-ins/metapost/render_metapost.c	2011-12-19 09:35:32.946501527 +0100
@@ -43,7 +43,7 @@
 #endif
 #include <errno.h>
 
-#include <glib/gstdio.h>
+#include <glib.h>
 
 #include "intl.h"
 #include "render_metapost.h"
--- dia-0.97.2/plug-ins/pgf/render_pgf.c.0002~	2011-01-07 13:11:34.000000000 +0100
+++ dia-0.97.2/plug-ins/pgf/render_pgf.c	2011-12-19 09:35:33.016501511 +0100
@@ -61,8 +61,8 @@ TODO:
 #endif
 #include <errno.h>
 
-#include <glib/gprintf.h>
-#include <glib/gstdio.h>
+#include <glib.h>
+#include <glib.h>
 
 #include "intl.h"
 #include "render_pgf.h"
--- dia-0.97.2/plug-ins/postscript/paginate_psprint.c.0002~	2009-11-07 15:28:34.000000000 +0100
+++ dia-0.97.2/plug-ins/postscript/paginate_psprint.c	2011-12-19 09:35:33.092501493 +0100
@@ -31,7 +31,7 @@
 #include <errno.h>
 #include <sys/stat.h>
 
-#include <glib/gstdio.h>
+#include <glib.h>
 
 #include "intl.h"
 #include "message.h"
--- dia-0.97.2/plug-ins/postscript/render_eps.c.0002~	2009-11-07 15:28:34.000000000 +0100
+++ dia-0.97.2/plug-ins/postscript/render_eps.c	2011-12-19 09:35:33.094501493 +0100
@@ -55,7 +55,7 @@
 #include <locale.h>
 #include <errno.h>
 
-#include <glib/gstdio.h>
+#include <glib.h>
 
 #include "intl.h"
 #include "render_eps.h"
--- dia-0.97.2/plug-ins/pstricks/render_pstricks.c.0002~	2011-01-07 13:11:34.000000000 +0100
+++ dia-0.97.2/plug-ins/pstricks/render_pstricks.c	2011-12-19 09:35:32.965501523 +0100
@@ -50,7 +50,7 @@ NOT WORKING (exporting macros):
 #endif
 #include <errno.h>
 
-#include <glib/gstdio.h>
+#include <glib.h>
 
 #include "intl.h"
 #include "render_pstricks.h"
--- dia-0.97.2/plug-ins/python/pydia-render.c.0002~	2009-11-07 15:28:34.000000000 +0100
+++ dia-0.97.2/plug-ins/python/pydia-render.c	2011-12-19 09:35:33.029501509 +0100
@@ -21,7 +21,7 @@
 
 #include <Python.h>
 #include <glib.h>
-#include <glib/gstdio.h>
+#include <glib.h>
 
 #include <locale.h>
 
--- dia-0.97.2/plug-ins/shape/shape-export.c.0002~	2009-11-08 12:14:56.000000000 +0100
+++ dia-0.97.2/plug-ins/shape/shape-export.c	2011-12-19 09:35:33.056501502 +0100
@@ -40,7 +40,7 @@
 #include <unistd.h>
 #endif
 
-#include <glib/gstdio.h>
+#include <glib.h>
 
 /* the dots per centimetre to render this diagram at */
 /* this matches the setting `100%' setting in dia. */
--- dia-0.97.2/plug-ins/svg/render_svg.c.0002~	2011-12-17 17:30:38.000000000 +0100
+++ dia-0.97.2/plug-ins/svg/render_svg.c	2011-12-19 09:35:33.103501491 +0100
@@ -30,7 +30,7 @@
 #endif
 
 #include <glib.h>
-#include <glib/gstdio.h>
+#include <glib.h>
 
 #include <libxml/entities.h>
 #include <libxml/tree.h>
--- dia-0.97.2/plug-ins/vdx/vdx-export.c.0002~	2009-12-27 17:22:38.000000000 +0100
+++ dia-0.97.2/plug-ins/vdx/vdx-export.c	2011-12-19 09:35:33.160501478 +0100
@@ -36,7 +36,7 @@
 #include <stdlib.h>
 #include <errno.h>
 #include <locale.h>
-#include <glib/gstdio.h>
+#include <glib.h>
 
 #include "intl.h"
 #include "message.h"
--- dia-0.97.2/plug-ins/vdx/vdx-import.c.0002~	2009-12-27 17:22:38.000000000 +0100
+++ dia-0.97.2/plug-ins/vdx/vdx-import.c	2011-12-19 09:35:33.207501466 +0100
@@ -28,7 +28,7 @@
 #include <string.h>
 #include <math.h>
 #include <glib.h>
-#include <glib/gstdio.h>
+#include <glib.h>
 #include <stdlib.h>
 #include <ctype.h>
 #include <libxml/tree.h>
--- dia-0.97.2/plug-ins/wpg/wpg.c.0002~	2009-11-07 15:28:34.000000000 +0100
+++ dia-0.97.2/plug-ins/wpg/wpg.c	2011-12-19 09:35:33.228501462 +0100
@@ -40,7 +40,7 @@
 #include <errno.h>
 
 #include <glib.h>
-#include <glib/gstdio.h>
+#include <glib.h>
 
 #include "intl.h"
 #include "message.h"
--- dia-0.97.2/plug-ins/xfig/xfig-export.c.0002~	2011-12-17 17:30:38.000000000 +0100
+++ dia-0.97.2/plug-ins/xfig/xfig-export.c	2011-12-19 09:35:33.267501452 +0100
@@ -16,7 +16,7 @@
 #include <locale.h>
 
 #include <glib.h>
-#include <glib/gstdio.h>
+#include <glib.h>
 
 #include "intl.h"
 #include "message.h"
--- dia-0.97.2/plug-ins/xfig/xfig-import.c.0002~	2009-11-07 18:13:53.000000000 +0100
+++ dia-0.97.2/plug-ins/xfig/xfig-import.c	2011-12-19 09:35:33.248501457 +0100
@@ -40,7 +40,7 @@
 #include <locale.h>
 
 #include <glib.h>
-#include <glib/gstdio.h>
+#include <glib.h>
 
 #include "intl.h"
 #include "message.h"
--- dia-0.97.2/plug-ins/xslt/xslt.c.0002~	2009-11-07 18:13:53.000000000 +0100
+++ dia-0.97.2/plug-ins/xslt/xslt.c	2011-12-19 09:35:33.084501495 +0100
@@ -27,7 +27,7 @@
 #include <string.h>
 #include <errno.h>
 
-#include <glib/gstdio.h>
+#include <glib.h>
 
 #include "filter.h"
 #include "intl.h"
--- dia-0.97.2/tests/test-boundingbox.c.0002~	2009-11-07 15:28:35.000000000 +0100
+++ dia-0.97.2/tests/test-boundingbox.c	2011-12-19 09:35:33.270501452 +0100
@@ -29,7 +29,7 @@
 #include <glib-object.h>
 
 #if GLIB_CHECK_VERSION(2,16,0)
-#include <glib/gtestutils.h>
+#include <glib.h>
 #endif
 #include "dialib.h"
 
--- dia-0.97.2/tests/test-objects.c.0002~	2009-11-07 15:28:35.000000000 +0100
+++ dia-0.97.2/tests/test-objects.c	2011-12-19 09:35:33.275501450 +0100
@@ -29,7 +29,7 @@
 #include <glib-object.h>
 
 #if GLIB_CHECK_VERSION(2,16,0)
-#include <glib/gtestutils.h>
+#include <glib.h>
 #endif
 
 #include "object.h"
